README.md

```diff
@@ -23,7 +26,8 @@ Low-level, unsafe Rust bindings for the [`capstone`][capstone] disassembly libra
 `capstone-sys` supports the following features:
 
 * `use_system_capstone`: use the system capstone instead of the bundled copy of the `capstone` library.
-    - Requires that `capstone` is already installed on the system
+    - Requires that `capstone` is already installed on the system. We highly recommend that you supply the exact version bundled with `capstone-sys`.
```

Comment:
What do you mean by supply the exact version here ? Maybe clarify slightly we only support 3. I think capstone doesn’t make breaking abi changes to header files iirc

Reply:
Perhaps my language was too strong--I agree that we should say we only support Capstone 3.

Reply:
It actually seems that Capstone will makes changes that could cause issues. Here are some changes made to Capstone 3.0 in the `include/` directory:
* [`33dd68b1ac`](https://github.com/aquynh/capstone/commit/33dd68b1ac4344cf39d551a1c7d620e9223bd7b6): change `unsigned int` to `x86_reg`
* [`cf731a27c2`](https://github.com/aquynh/capstone/commit/cf731a27c2bffc000bc3f4ba7d15f819d87014a1): improve type safety, use ppc_reg instead of unsigned int
* [`375d7432c8`](https://github.com/aquynh/capstone/commit/375d7432c80675e6ce4ba46519776cab1018009b): Rename CS_OPT_NONE to CS_OPT_INVALID
* [`f585a84966`](https://github.com/aquynh/capstone/commit/f585a84966f8f5de951705d746b264900ba205c5): add CS_VERSION_{MAJOR, MINOR, EXTRA}
* [`2fa4744f1d`](https://github.com/aquynh/capstone/commit/2fa4744f1dc46602f6668e70d0bc79bb118e41b7): arm64: rename enum arm64_mrs_reg to arm64_sysreg

I think the existing warning should stand. Using a later or earlier version of Capstone *could* cause issues.

Reply:
Okie, sounds good to me